apps/condo/domains/meter/components/TabContent/MeterReading.tsx

```diff
@@ -262,7 +255,7 @@ const MeterReadingsTableContent: React.FC<MetersTableContentProps> = ({
                                     <Row justify='start' gutter={FILTERS_CONTAINER_GUTTER} style={{ flexWrap: 'nowrap' }}>
                                         <Col style={QUICK_FILTERS_COL_STYLE}>
                                             <DateRangePicker
-                                                value={dateRange || dateFallback}
+                                                value={dateRange}
```

Comment:
Why do you need to remove default value of the filter?

Reply:
designer proposed this during design review because when user initially visits the page and if they only have readings older than a week -- they won't see anythin and design review decided it's misleading